database.cc

```diff
@@ -1644,6 +1644,27 @@ operator<<(std::ostream& os, const atomic_cell& ac) {
     return os << atomic_cell_view(ac);
 }
 
+std::ostream&
```

Comment:
It got there because the non-type-aware printer for atomic_cell is there. I'll add a patch that moves the existing printer to atomic_cell.cc, and then this will naturally go there as well.